### ISSUE TITLE
Requesting detailed documentation regarding what causes NVDA to announce the word 'document'

### ISSUE THREAD

**Issue by soehlerSF** [nvaccess/nvda#18696]
### Is your feature request related to a problem? Please describe.
No problem in particular: we are a software-as-a-service that provides capabilities for customers to create their own lightning components. Sometimes customers will configure a component that, when tabbed to (or focused), NVDA will read out 'document'. We're unable to provide a reproduction of this, but we suspect that it has to do with the presence of the html element <article> in relation to other elements where focus is passing from one article to another. We're unable to reproduce this consistently.  

### Describe the solution you'd like
We're hoping NVDA can update their documentation to indicate under what specific circumstances 'document' is read aloud, with emphasis on what configuration causes this to occur and with what elements. The reason this is important is because customers are comparing this product with the functionality of MacVO (for OS and browser consistency in testing) and MacVO does not seem to announce document focus swaps like NVDA does, so a better understanding of what technically causes this announcement would be of great help.

### Describe alternatives you've considered
n/a

### Additional context
n/a
